lib/numberpicker.dart

```diff
@@ -170,26 +180,41 @@ class NumberPicker extends StatelessWidget {
       child: new Container(
         height: _listViewHeight,
         width: listViewWidth,
-        child: new ListView.builder(
-          controller: intScrollController,
-          itemExtent: itemExtent,
-          itemCount: itemCount,
-          cacheExtent: _calculateCacheExtent(itemCount),
-          itemBuilder: (BuildContext context, int index) {
-            final int value = _intValueFromIndex(index);
-
-            //define special style for selected (middle) element
-            final TextStyle itemStyle =
-            value == selectedIntValue ? selectedStyle : defaultStyle;
-
-            bool isExtra = index == 0 || index == itemCount - 1;
-
-            return isExtra
-                ? new Container() //empty first and last element
-                : new Center(
-              child: new Text(value.toString(), style: itemStyle),
-            );
-          },
+        child: Stack(
+          children: [
+            new ListView.builder(
+              controller: intScrollController,
+              itemExtent: itemExtent,
+              itemCount: itemCount,
+              cacheExtent: _calculateCacheExtent(itemCount),
+              itemBuilder: (BuildContext context, int index) {
+                final int value = _intValueFromIndex(index);
+
+                //define special style for selected (middle) element
+                final TextStyle itemStyle =
+                ((value == selectedIntValue) && highlightSelectedValue)
+                    ? selectedStyle
+                    : defaultStyle;
+
+                bool isExtra = index == 0 || index == itemCount - 1;
+
+                return isExtra
+                    ? new Container() //empty first and last element
+                    : new Center(
+                  child: new Text(value.toString(), style: itemStyle),
+                );
+              },
+            ),
+            new Center(
```

Comment:
Could you extract it to a standalone widget in the same file? Like `NumberpickerDecorationBox`?